lib/ProductOpener/Display.pm

```diff
@@ -10992,7 +10992,10 @@ sub display_nested_list_of_ingredients ($ingredients_ref, $ingredients_text_ref,
 		${$ingredients_list_ref}
 			.= "<li>" . "<span$class>" . $ingredient_ref->{text} . "</span>" . " -> " . $ingredient_ref->{id};
 
-		foreach my $property (qw(origin labels vegan vegetarian from_palm_oil percent_min percent percent_max)) {
+		foreach my $property (
+			qw(origin labels vegan vegetarian from_palm_oil percent_min percent percent_max ciqual_proxy_food_code ciqual_food_code)
```

Comment:
Good, can you change the order a bit:

```suggestion
			qw(origin labels vegan vegetarian from_palm_oil ciqual_food_code ciqual_proxy_food_code percent_min percent percent_max)
```

Reply:
Initial demand was to display it only for moderators. I don't know if it's important (@teolemon ?)